server/config/config.go

```diff
@@ -114,6 +114,15 @@ type KinesisConfig struct {
 	ResultStream     string `yaml:"result_stream"`
 }
 
+// S3Config defines config to enable file carving storage to an S3 bucket
+type S3Config struct {
+	Bucket           string
+	Prefix           string
+	AccessKeyID      string `yaml:"access_key_id"`
+	SecretAccessKey  string `yaml:"secret_access_key"`
+	StsAssumeRoleArn string `yaml:"sts_assume_role_arn"`
```

Comment:
At some point I think we'll want to find a way to make the AWS configuration shared across the various AWS plugins. This may be something I should take on and plan for changing the config structure in 4.0.